plugins/woocommerce/phpstan-baseline.neon

```diff
@@ -15655,6 +15656,8 @@ parameters:
 			path: includes/class-wc-cart-session.php
 
 		-
+=======
+>>>>>>> 31360e0002 ([Cart shortcode] Fix undo cart item removal action (#62472))
```

Comment:
Need to remove the merge conflict stuff.

Reply:
fixed in 7aba6bc0d0b0258b9c6487039e23a1b259afa4fc